geographic_info/package.xml

```diff
@@ -1,7 +1,8 @@
-<package>
-
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>geographic_info</name>
-  <version>0.5.2</version>
+  <version>0.5.3</version>
```

Comment:
You should at least jump a minor version if not a major version to leave space for a potential maintenance releases in the existing series.

Since this is relatively mature and not likely to change a lot I think it's reasonable to call it a 1.0.0

Reply:
Can do